src/tests/verified_chats.rs

```diff
@@ -43,7 +43,7 @@ async fn check_verified_oneonone_chat(broken_by_classical_email: bool) {
             b"Subject: Re: Message from alice\r\n\
           From: <bob@example.net>\r\n\
           To: <alice@example.org>\r\n\
-          Date: Mon, 12 Dec 2022 14:33:39 +0000\r\n\
+          Date: Mon, 12 Dec 3000 14:33:39 +0000\r\n\
```

Comment:
This message doesn't have the Autocrypt header, so it doesn't matter if it's from the past or from the future, but let's choose the latter to make sure even that doesn't break contact verification